csrc/quantization/compressed_tensors/int8_quant_kernels.cu

```diff
@@ -48,12 +67,13 @@ void static_scaled_int8_quant(torch::Tensor& out,    // [..., hidden_size]
   int num_tokens = input.numel() / hidden_size;
   dim3 grid(num_tokens);
   dim3 block(std::min(hidden_size, 1024));
+  const float inverted_scale = 1.0f / scale;
```

Comment:
divide this on host so that we can use multiplication instructions instead of division in the kernel itself. Value is constant so works perfectly fine and simply.